py/torch_tensorrt/fx/trt_module_next.py

```diff
@@ -0,0 +1,45 @@
+from operator import truediv
+from typing import Any, List, Sequence
+
+import torch
+
+from torch.classes.tensorrt import Engine
+from torch.ops.tensorrt import execute_engine
+
+from torch_tensorrt import (_C, Device)
+
+class TRTModule(torch.nn.module):
```

Comment:
Is this some staging module?

Reply:
Yeah this would be an alternate to the existing TRTModule so it is opt-in for users for the next release for evaluation.